src/components/PicSlider/caption.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
-const Caption = (props) => {
+const Caption = (props: { caption: any }) => {
```

Comment:
shouldn't this be of type string here?

Reply:
Yeah it should. Copied over from CollegeLutheran without thinking about that.